### PR TITLE
Clarify task sequence step options/conditions

### PR DESCRIPTION
**Pull request by ChadSimmons**
- Update task sequence step options/conditions to reflect the wording in the UI
- make the descriptions more consistent
- clarify that the existence of file, folder, and registry key can be evaluated

### CODE REVIEW

**Comment by BalaDelli** [comment]
#sign-off

File: memdocs/configmgr/osd/understand/task-sequence-editor.md [hunk 1]
```diff
@@ -152,24 +152,24 @@ The following types of conditions are available:
...
 
-- **OS version**: Evaluate the OS version of the device where the task sequence runs. This list is the general OS versions used throughout Configuration Manager. To evaluate a more detailed OS version, such as a specific version of Windows 10, use the **Query WMI** condition.
+- **Operating System Version**: Evaluate the OS version of the device where the task sequence runs. This list is the general OS versions used throughout Configuration Manager. To evaluate a more detailed OS version, such as a specific version of Windows 10, use the **Query WMI** condition.
 
-- **OS language**: Evaluate the OS language of the device where the task sequence runs. This list includes the 257 languages that Windows supports.
+- **Operating System Language**: Evaluate the OS language of the device where the task sequence runs. This list includes the 257 languages that Windows supports.
 
-- **File properties**: Evaluate the version or timestamp of any file on the device where the task sequence runs.
+- **File Properties**: Evaluate the existence, version or timestamp of any file on the device where the task sequence runs.
```
Comment (JohanFreelancer9) [inline memdocs/configmgr/osd/understand/task-sequence-editor.md:164]:
```suggestion
- **File Properties**: Evaluate the existence, version, or timestamp of any file on the device where the task sequence runs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ChadSimmons, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ChadSimmons** [comment]
@JohanFreelancer9, can you please help me understand what is missing to get this PR merged?  Thanks!

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9, can you please help me understand what is missing to get this PR merged? Thanks!

@BalaDelli, can you please help to get this PR merged? Many thanks.

**Review by BalaDelli — APPROVED** [review]

**Comment by BalaDelli** [comment]
#sign-off

**Comment by dougeby** [comment]
This was signed-off on but not merged. Merging.

**Pull request merged**